### DISCUSSION TITLE
GTR4 notifications are stuck on the phone

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#205] [Developer Feedback]
Once we received OS 2.1 (FW: 3.23.3.1) **action button** that allows us to make notification read (Delete most of the time) on the watch does not remove it on the phone, yet it used to do so since release day of GTR4.
![image](https://github.com/zepp-health/zeppos-forum/assets/1674725/ca9a982d-0fa0-4883-ac51-f0912621d957)

I asked other users, no matter android version or phone manufactur, it just stopped working in latest FW.
@MyoungXUE @liqin-ZeppHealth @htoooth

**Comment by liqin-ZeppHealth** [comment]
hello @zdunex25 , thanks for you feedback, we are confirming the details of this feature with our colleagues and will get back to you later with a further answer.

**Comment by liqin-ZeppHealth** [comment]
Hi, it is confirmed that this feature has been adjusted on the latest firmware. Previously on Android it was synced to the phone side of the notification status (iOS does not support it), but on the latest version it is not even synced to the phone side. If you think this feature is helpful, you can submit a user feedback on the Zepp app， thanks

**Reply by zdunex25** [reply]
Is it going to be possible through Zepp app update? Will not it break other devices such as trex2 or gtr3 with older implementation in firmware?

**Reply by aredon** [reply]
Any update on this?
